recipe/meta.yaml

```diff
@@ -36,13 +30,27 @@ requirements:
   run:
     - python
 
+{% set skip_tests = [
+    [aarch64 or ppc64le, 'test_loss_of_precision',              'Seems to fail on aarch64 and ppc64le'],
```

Comment:
@hmarrfk, for example, `test_loss_of_precision` is skipped for `aarch64 or ppc64le`

Reply:
I think I was confused by the conda forge keyword. I guess you needed to add something in case the statement was empty